forui/lib/src/foundation/portal/portal.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:forui/forui.dart';
+
+/// A portal renders a follower widget that "floats" on top of a target widget.
+///
+/// Similar to an [OverlayPortal], it requires an [Overlay] ancestor. Unlike an [OverlayPortal], the follower is aligned
+/// relative to the target.
+///
+/// See:
+/// * [FPortalFollowerShift] for the various follower shifting strategies.
+/// * [OverlayPortalController] for controlling the follower's visibility.
+/// * [OverlayPortal] for the underlying widget.
+class FPortal extends StatefulWidget {
+  /// The controller that shows and hides the follower. It initially hides the follower.
+  final OverlayPortalController controller;
+
+  /// The anchor of the follower to which the [childAnchor] is aligned to. Defaults to [Alignment.topCenter].
+  final Alignment followerAnchor;
+
+  /// The anchor of the target to which the [followerAnchor] is aligned to. Defaults to [Alignment.bottomCenter].
+  final Alignment childAnchor;
```

Comment:
What's the difference between a `followerAnchor` and `childAnchor`? Also it seems like `childAnchor` becomes `targetAnchor` in the `_Alignment` widget.

Reply:
Ah it's a slip up, I renamed it in the `FPopover` PR.